src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -0,0 +1,13 @@
+Hello ${userName},
```

Comment:
@wkurniawan07 this email should not go out before the grace period is over. Is that taken care of?